src/evaluations/evaluator.py

```diff
@@ -224,38 +237,94 @@ def __init__(self, evaluation_config, sketch_estimator_config_list, run_name,
     self.scenario_random_states = scenario_random_states
 
   def __call__(self):
-    self.evaluate_all()
-
-  def evaluate_all(self):
-    """Evaluate all estimators under all scenarios."""
+    self.evaluate_all_parallel()
+
+  def evaluate_all_parallel(self):
```

Comment:
Should there be some tests added?

Reply:
This function has the same output as the previous evaluate_all function, so I'm just using those tests. If I see a need for extra unit tests, I will add them in the future, but for now the API and tests are remaining the same.